### PR TITLE
[STAKAN-3] New authorization flow

### PR DESCRIPTION
**Pull request by AndrewKhizhnyak**
### New authorization flow 

- Made /features public 
- Showed list of available networks
- Moved BlockchainNetworkId from stkr-jssdk to types
- Added avalanche networks id
- Now Header button leads to /features 
- Feature items available on /features page
- Implemented a new design access request screen
- Staker in the header now leads to /features
- Connect via unsupported blockchain leads to the new component
- Implemented a new component for unsupported networks according to the design

Screenshot 2021-06-23 at 12 10 21

[task](https://ankrnetwork.atlassian.net/browse/STAKAN-3)

[design](https://www.figma.com/file/mQ9yM5xfy57NASyuT8ZG0z/Stkr-Transform-ANKR?node-id=2153%3A21)

### CODE REVIEW

File: src/modules/features/FeaturesList.tsx [hunk 1]
```diff
@@ -55,7 +58,6 @@ export const FeaturesList = () => {
             ]}
             buttonText={t('features-list.action.stake-eth.mainnet')}
             onClickTo={STAKER_DASHBOARD_PATH}
-            disabled={!isEthStakingAvailable}
```
Comment (Kud8) [inline src/modules/features/FeaturesList.tsx:58]: а зачем эти проверки убрал?

File: src/modules/layout/components/Header/HeaderComponent.tsx [hunk 2]
```diff
@@ -69,15 +72,17 @@ export const HeaderComponent = ({
     />
   );
 
-  const renderedAppButton = (
+  const renderedAppButton = isFeatures ? (
+    <div />
```
Comment (Kud8) [inline src/modules/layout/components/Header/HeaderComponent.tsx:76]: что за пустой div?)

File: src/modules/features/FeaturesList.tsx (hunk 1 above)
Comment (AndrewKhizhnyak) [reply src/modules/features/FeaturesList.tsx:58]: I removed these checks coz in a new authorization flow u need to click these buttons to sign in

File: src/modules/layout/components/Header/HeaderComponent.tsx (hunk 2 above)
Comment (AndrewKhizhnyak) [reply src/modules/layout/components/Header/HeaderComponent.tsx:76]: That was synced with @BigBadAlien. This solution coz of grids in the header, without this block layout breaks

File: src/Routes.tsx [hunk 3]
```diff
@@ -274,26 +276,41 @@ export function Routes() {
...
-      <PrivateRoute
+      <Route exact path={FEATURES_PATH} component={FeaturesListContainer} />
+
+      <GuardRoute
         path={STAKER_DASHBOARD_PATH}
         component={StakerDashboardContainer}
-        exact={true}
+        exact
+        availableNetworks={[
+          BlockchainNetworkId.mainnet,
```
Comment (BigBadAlien) [inline src/Routes.tsx:286]:
```suggestion
    isMainnet ? BlockchainNetworkId.mainnet : BlockchainNetworkId.goerli
```

let's not expose Goerli in UI for prod users

File: src/UiKit/GuardRoute.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+export interface INetwork {
+  title: string;
+  icon: ReactNode;
+  chainId: BlockchainNetworkId;
+}
+
+const networks: INetwork[] = [
+  {
+    title: 'Ethereum Mainnet',
```
Comment (BigBadAlien) [inline src/UiKit/GuardRoute.tsx:44]: All literal must be translated.

Here `useLocaleMemo` is required

File: src/store/reducers/userReducer.ts [hunk 5]
```diff
@@ -32,17 +33,19 @@ const initialState: IUserState = {
   isConnected: false,
   isConnectionAvailable: false,
   locale: Locale.en,
+  chainId: NaN,
```
Comment (BigBadAlien) [inline src/store/reducers/userReducer.ts:36]: Let's use Soviet undefined here

**Review by BigBadAlien — APPROVED** [review]

**Review by Kud8 — APPROVED** [review]

**Pull request merged**